API/Controllers/ProjectController.cs

```diff
@@ -188,7 +188,7 @@ public async Task<IActionResult> UpdateProject(int projectId, [FromBody] Project
                     Detail = "The user is not allowed to edit the project.",
                     Instance = "2E765D18-8EBC-4117-8F9E-B800E8967038"
                 };
-                return BadRequest(problem);
+                return Unauthorized(problem);
```

Comment:
The 401 Unauthorized status should be used when the user is not authenticated / signed in.
In this case, the user is authenticated but does not have the right permissions. The return status should be 403 Forbidden.

```suggestion
                return Forbidden(problem);
```